daemon/src/rollover_maker.rs

```diff
@@ -177,6 +177,27 @@ impl Actor {
 
         let taker_id = self.taker_id;
 
+        // the maker computes the rollover fee and sends it over to the taker so that both parties
+        // are on the same page
+        let complete_fee = match rollover_params.version {
```

Comment:
🛠 I don't think this distinction makes sense if you change the protocol to the maker coming up with the fee. If the maker comes up with the fee the fee should *always* be the complete amount, `RolloverVersion::V1` does not make sense here.

Reply:
Can do.

Reply:
see [`fa976dd` (#1981)](https://github.com/itchysats/itchysats/pull/1981/commits/fa976dd8754345c9e9616bc5821bda3f253eb449)